iamchrissmith/uncorked
# Searching

#### What does this PR do?
- adds autocomplete searching and an API endpoint for wine names
#### Where should the reviewer start?
- `api/v1/wines_controller.rb`
#### How should this be manually tested?
- going to the wines page and starting to type in the wine search box and see magic
#### Any background context you want to provide?

#### What are the relevant tickets?
completes: [#149039657](https://www.pivotaltracker.com/story/show/149039657)
completes one endpoint for: [#149116771](https://www.pivotaltracker.com/story/show/149116771)
#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran?
no
  - Do Environment Variables need to be set?
no
  - Any other deploy steps?
no
